# Generated by Django 3.1.5 on 2021-01-22 14:35

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('app1', '0034_auto_20210122_1916'),
    ]

    operations = [
        migrations.AlterField(
            model_name='with',
            name='cause_tag',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='withes', to='app1.causetag'),
        ),
        migrations.CreateModel(
            name='LatestWith',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('cause_tag', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='app1.causetag')),
                ('latest_with', models.OneToOneField(null=True, on_delete=django.db.models.deletion.PROTECT, to='app1.with')),
                ('problem', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='app1.problem')),
                ('site_user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
